### ISSUE TITLE
Add  sound effects

### ISSUE THREAD

**Issue by deepak2431** [vineetjc/pygame-Coin-Fall-#15]

**Comment by amrit-choudhary** [comment]
@deepak2431 have you started or should i work on this?

**Comment by vineetjc** [comment]
Seeing the above I'm assigning @deepak2431

**Comment by vineetjc** [comment]
#49 adds bg music, can be improved by adding more sound effects, mute/unmute, volume, toggle sfx options etc.

**Comment by deepak2431** [comment]
Yeah I am working on that too.Bit I am busy because of my college work so I can't get much time.I am trying to add more sound effects as after setting will be updated I would add mute music option there.And I am trying to add coin pickup sound.

**Comment by deepak2431** [comment]
@vineetjc You have rename the Music folder name to music you need to change the too in sound_manager.py as music while loading the music file.Shall I make a pull request regarding it or you would work on that?

**Comment by vineetjc** [comment]
I'll make that change @deepak2431 well spotted
